src/OSPSuite.Assets.Images/ApplicationIcons.cs

```diff
@@ -47,6 +47,8 @@ public static class ApplicationIcons
       public static readonly ApplicationIcon BoxWhiskerAnalysisRed = AddNamedIcon("BoxWhiskerAnalysisRed");
       public static readonly ApplicationIcon Brain = AddNamedIcon("Brain");
       public static readonly ApplicationIcon BuildingBlockExplorer = AddNamedIcon("BuildingBlockExplorer");
+      public static readonly ApplicationIcon ModuleExplorer = AddNamedIcon("ModuleExplorer"); // TODO add an icon svg file. Presently just a copy of BBExplorer svg
```

Comment:
No. Add at the end of the list otherwise it will messup the layout saved in devexpress